<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhereIsSafeDockWidgetBase</class>
 <widget class="QDockWidget" name="WhereIsSafeDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>788</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>788</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Where Is Safe</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#Screen{
background-image: url(:/graphics/Phone_Template.png);
background-position: top left;
}

QFrame#FirstPage{
background-image: url(:/graphics/first_page.png);
background-position: top left;
}

QCommandLinkButton#start_btn{
border: none;
font-size: 12pt;
color: red;
}

QCommandLinkButton#backm_btn{
border: none;
font-size: 12pt;
color: blue;
}

QCommandLinkButton#backp_btn{
border: none;
font-size: 12pt;
color: blue;
}

QPushButton#endcall_btn{
background-color:transparent;
background-image: url(:/graphics/endcall.png);
background-position: top left;
}

QPushButton#profile_btn{
background-image: url(:/graphics/profile.png);
background-position: top left;
}

QPushButton#monitor_btn{
background-image: url(:/graphics/monitor.png);
background-position: top left;
}

QPushButton#call_btn{
background-image: url(:/graphics/call112.png);
background-position: top left;
}

QFrame#Call112{
background-image: url(:/graphics/callpage.png);
background-position: top left;
}

QFrame#Profile{
background-image: url(:/graphics/profilepage.png);
background-position: top left;
}
</string>
   </property>
   <widget class="QFrame" name="Screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>721</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="App">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>721</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QgsMapCanvas" name="map_canvas" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="danger">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>690</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="safe">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>690</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="profile_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="monitor_btn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="call_btn">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="Monitor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCommandLinkButton" name="backm_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>507</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to the map</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="FirstPage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCommandLinkButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>351</width>
        <height>121</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>WARNING! 
There is a chemical spread near
you. check for more info </string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Call112">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="endcall_btn">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>470</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="Profile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>591</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="yes">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="no">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="child">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Below 14 years old</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="young">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>14 - 60 years old</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="old">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Over 60 years old</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="walk">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Walking</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="bike">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>410</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bike</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="car">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Car</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="backp_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>510</y>
        <width>225</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to the map</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Does your physique allow you to travel to a safe place?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>How old are you?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>What is your travel mode?</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QWidget</extends>
   <header location="global">qgis.gui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
